app/controllers/cars_controller.rb

```diff
@@ -1,11 +1,12 @@
 class CarsController < ApplicationController
   def index
     @cars = Car.all
+  end
 
-    # if params[:search]
-    #   @cars = Car.search(params[:search])
-    # else
-    #   @cars = Car.all
-    # end
+  def destroy
+    car = Car.find(params[:id])
+    car.destroy
+    flash[:deleted] = 'You have removed the car.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.